core/fortran-example/tutorial/02_tuple/main.cpp

```diff
@@ -0,0 +1,107 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+
+#include <iostream>
+
+//---------------------------------------------------------------------------//
+// Tuple example.
+//---------------------------------------------------------------------------//
+
+    /*
```

Comment:
The indentation looks weird here

Reply:
Instead of copy-pasting these comments from the C++ tutorial let's update them to reflect what this Fortran example is doing.

Reply:
This part is the same as in the c++ tutorial. I will keep it as it does serve the purpose here (and people don't have to look for the c++ tutorial to see the comments). I have put some explanation for the Fortran part of the code.